domain/src/main/java/com/pocs/domain/usecase/user/GetMyUserInfoUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.pocs.domain.usecase.user
+
+import com.pocs.domain.model.user.UserDetail
+import com.pocs.domain.repository.UserRepository
+import javax.inject.Inject
+
+class GetMyUserInfoUseCase @Inject constructor(
```

Comment:
usecase 이름을 레파지토리의 이름과 통일시키는게 좋아보이네요
```suggestion
class GetCurrentUserDetailUseCase @Inject constructor(
```